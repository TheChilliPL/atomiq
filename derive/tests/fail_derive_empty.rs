#![cfg(not(clippy))]

use atomiq_derive::Atomizable;

#[derive(Atomizable)]
struct TestStruct;

fn main() {}