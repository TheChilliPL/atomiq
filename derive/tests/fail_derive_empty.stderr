error: Atomizable can only be derived for structs with a single field.
 --> tests/fail_derive_empty.rs:3:10
  |
3 | #[derive(Atomizable)]
  |          ^^^^^^^^^^
  |
  = note: this error originates in the derive macro `Atomizable` (in Nightly builds, run with -Z macro-backtrace for more info)
